src/assets/scss/2-layout/1-header.scss

```diff
@@ -0,0 +1,28 @@
+header {
+  padding: 20px 0px;
```

Comment:
Nitpick: consider having several different spacing variables and reuse it to build consistent spacing.
Multiples of 4 work nicely. 😃